### PR TITLE
Bash 03 hw task 1 fix owner

### PR DESCRIPTION
**Pull request by ghostzero2000**
Implement internal logic fix_owner.sh

### CODE REVIEW

**Comment by yekovalyov** [comment]
This script doesn't request an action without --force option.

File: 03-Bash/fix_owner/fix_owner.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+! #bin/bash
+
+echo "test fix_owner.sh"
```
Comment (yekovalyov) [inline 03-Bash/fix_owner/fix_owner.sh]: What is the purpose of this file?

Comment (ghostzero2000) [reply 03-Bash/fix_owner/fix_owner.sh]: I created an empty file for the test. What should I do with this commit? Should I squash it?

Comment (yekovalyov) [reply 03-Bash/fix_owner/fix_owner.sh]: For these tasks the requirement was 1 task = 1 commit. Force pushes to your private branches are allowed.

**Pull request merged**